routers/web/repo/view.go

```diff
@@ -720,6 +721,7 @@ func Home(ctx *context.Context) {
 	}
 
 	ctx.Data["FeedURL"] = ctx.Repo.Repository.HTMLURL()
+	ctx.Data["CanPin"] = ctx.IsSigned && user_service.CanPin(ctx, ctx.Doer, ctx.Repo.Repository)
```

Comment:
```suggestion
	ctx.Data["CanPinRepos"] = ctx.IsSigned && user_service.CanPin(ctx, ctx.Doer, ctx.Repo.Repository)
```
Thinking of something like issue pinning, I think it would be beneficial to add **what** you can pin.